controllers/sriovoperatorconfig_controller.go

```diff
@@ -106,6 +106,44 @@ func (r *SriovOperatorConfigReconciler) Reconcile(ctx context.Context, req ctrl.
 		return reconcile.Result{}, err
 	}
 
+	// Fetch the SriovNetworkNodePolicyList
+	policyList := &sriovnetworkv1.SriovNetworkNodePolicyList{}
```

Comment:
The creation of policy is handled by sriovnetworknodepolicy controller i think we can rely on that controller to create the policy.

perhaps we need to fail if policy is not found in sriovnetworknodepolicy controller Reconcile call

Reply:
we do it at line ##120

Reply:
returning no error and empty reconcile will not requeue, we should requeue here imo to let node policy controller create the default policy.